src/pages/RepoPage/ComponentsTab/BackfillBanners/hooks.spec.tsx

```diff
@@ -11,7 +12,7 @@ const queryClient = new QueryClient({
 })
 
 const wrapper =
-  (initialEntries = '/gh/codecov/test') =>
+  (initialEntries = '/gh/codecov/test'): React.FC<PropsWithChildren> =>
```

Comment:
I would recommend doing `React.PropsWithChildren` that way you don't need an import.